ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -242,7 +250,10 @@ class TFLiteCore
 
   TFLiteInterpreter *interpreter;
   TFLiteInterpreter *interpreter_sub;
-
+  
+  GHashTable *shared_table;
```

Comment:
static?

Reply:
The hash table is created and managed at `tensor_filter_common.c`. This pointer is just existed to access the table. The reason why `filter_tf_lite` should access the table is to find the shared interpreter and remove it when the filter is destroyed.